scss/detail.scss

```diff
@@ -93,5 +93,97 @@
         }
       }
     }
+  }  
+  @media only screen and (max-width: $flex-mobile) {
+    height: 667px;
+    width: 100%;
+    flex-direction: column-reverse;
+  
+    .return-button {
+      align-self: center;
+      font-size: 18px;
+      line-height: 24px;
+      margin: 20px 0 0 0;
+
+      &::before {
+        margin-right: 12px;
+      }
+    }
+
+    .cover-text-container {
+      flex-direction: column;
+      align-items: center;
+      height: 570px;
+      width: 336px;
+
+      .cover-page-badge-container {
+        margin-bottom: 21px;
+        
+        .cover-page {
+          height: 299px;
+          width: 212px;
+          margin: 38px 0 0 0;
+        }
+        
+        .badge {
+          $width: 95px;
+          width: calc(#{$width} * 0.7);
+          $height: 119px;
+          height: calc(#{$height} * 0.7);
+          right: -16px;
+          top: 27px;
+        }
+      }
+
+      .container-text {
+        display: flex;
+        flex-direction: column;
+        height: 570px;
+        width: 336px;
+        
+        .container-book {
+          display: flex;
+          justify-content: flex-start;
+          width: 300px;
+          margin: 0 0 0 0;
+          border-bottom: 4px solid $earls-green;
+          margin-left: 16px;
```

Comment:
There are 2 rules regarding "margin", with the shorthand you can achieve the same result while doing it in one line (assigning 16px to the left and set everything else in 0)
```scss
margin: 0 0 0 16px;
```